packages/pyright-internal/src/readonlyAugmentedFileSystem.ts

```diff
@@ -161,19 +167,28 @@ export class ReadOnlyAugmentedFileSystem implements FileSystem {
         return this.realFS.isInZip(path);
     }
 
-    protected recordMovedEntry(mappedPath: string, originalPath: string, reversible = true, isFile = true) {
+    protected recordMovedEntry(mappedPath: string, originalPath: string, rootPath: string) {
         this._entryMap.set(mappedPath, originalPath);
-
-        if (reversible) {
-            this._reverseEntryMap.set(originalPath, mappedPath);
-        }
+        this._reverseEntryMap.set(originalPath, mappedPath);
 
         const directory = ensureTrailingDirectorySeparator(getDirectoryPath(mappedPath));
         const folderInfo = getOrAdd(this._folderMap, directory, () => []);
 
         const name = getFileName(mappedPath);
         if (!folderInfo.some((entry) => entry.name === name)) {
-            folderInfo.push({ name, isFile });
+            folderInfo.push({ name, isFile: true });
+        }
+
+        // Add the directory entries for the sub paths as well.
+        const subPathEntries = getRelativePathComponentsFromDirectory(rootPath, mappedPath, false);
```

Comment:
give directory instead of `mappedPath` ? since you already got one?

Reply:
`false` means `case insensitive`? shouldn't we give it based on file system's case sensitivity?

Reply:
so, talked with Rich. `false` is okay since `mappedPath` is always `rootPath` + `relative Path to the stub file`, so casing on `rootPath` part should always match.